CppWinRTProjectionSample/SampleConsoleApp/SampleConsoleApp.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
```

Comment:
it would be great to make all Windows SDK versions (e.g., 10.0.19041.0) defined in the root directory.build.props, so we can upgrade everything at once.  maybe open a separate issue for that.

Reply:
opened separate issue